### ISSUE TITLE
Potential error in TFT balance amount required in wallet to get discount on dedicated nodes.  Balance requirement too low?

### ISSUE THREAD

**Issue by weynandkuijpers** [threefoldtech/test_feedback#306]

**Comment by weynandkuijpers** [comment]
Hi - any update on this?

**Comment by weynandkuijpers** [comment]
Also:
<img width="338" alt="image" src="https://user-images.githubusercontent.com/13766992/199668307-3034c44d-cac2-4a47-a2ee-980d349129a3.png">

The [wiki states 70% discount](https://library.threefold.me/info/manual/#/manual__dashboard_portal_dedicated_nodes?id=billing-amp-pricing) on the retail price is you rent a dedicated node. The percentage is is not displayed in the pop-up

**Comment by ramezsaeed** [comment]
for the percentage in the comment, its fixed but not yet deployed on mainnet.
will verify with the docs ..

**Comment by A-Harby** [comment]
Yes, that would be true, just for more clarification based on the [Discount Levels](https://library.threefold.me/info/threefold/#/tfgrid/grid/pricing?id=discount-levels), And using the resource available on node 7 as an example, it should cost $17.86/month for renting the node with no TFT balance. 
![image](https://user-images.githubusercontent.com/109473873/199696711-f9b6e74a-5924-4f1c-83cd-8dd781562e2d.png)
Therefor for having 40% discount is having a balance enough for renting this node for 12 months which will be having $214.32 (7144 TFT), and having 400 TFT only is not even enough for having the default package. 
![image](https://user-images.githubusercontent.com/109473873/199698643-381ba15e-1126-422e-a478-d225d615fc3e.png)

**Comment by AhmedHanafy725** [comment]
I think the dashboard calculates the discount from the balance as USD not TFT, @Omarabdul3ziz can you check?
